### ISSUE TITLE
Any detailed documents about this package?

### ISSUE THREAD

**Issue by weixsong** [uber/horovod#51]
Hi,

About how to use the parameters, how to save model and restore model, is there any detailed documents about this package?
Such as code beblow:
    hooks = [hvd.BroadcastGlobalVariablesHook(0),

             tf.train.StopAtStepHook(last_step=10000),

             tf.train.LoggingTensorHook(tensors={'step': global_step, 'loss': loss},

                                        every_n_iter=10),

             ]

**Comment by alsrgv** [comment]
Hi @weixsong,

The question you ask about parameters, saving and restored model are generally answered on the official TensorFlow [website](https://www.tensorflow.org/).

If you use `tf.train.MonitoredTrainingSession()` ([doc](https://www.tensorflow.org/api_docs/python/tf/train/MonitoredTrainingSession)), you can specify the parameter `checkpoint_dir` and model will be restored/checkpointed into that directory.

It may be a good idea to save checkpoint only on worker 0. To do that, you can pass another parameter `save_checkpoint_secs=600 if hvd.rank() == 0 else 100000000000000000`.

**Comment by weixsong** [comment]
@alsrgv , thanks.

**Comment by weixsong** [comment]
@alsrgv , I'm using the following code:
    is_chief = (hvd.rank() == 0)
    save_secs = 60 if is_chief else 800000000000000000
    with tf.train.MonitoredTrainingSession(hooks=hooks,
                                           config=config,
                                           is_chief=is_chief,
                                           checkpoint_dir=args.output_model_path,
                                           save_checkpoint_secs=save_secs) as mon_sess:

But the TF blocked by "Waiting for model to be ready" very long time, do you know how to fix this issue?

**Comment by alsrgv** [comment]
Yes, for Horovod all the workers should be "chiefs", as they all should initialize variables and do other initialization procedures. I'll update the example to explicitly clarify this - thanks for your feedback!

**Comment by weixsong** [comment]
thanks.
